aws/ecs/lib/stack-config.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ Copyright (c) 2016-2020, CKSource - Frederico Knabben. All rights reserved.
+ */
+
+import { LaunchType } from '@aws-cdk/aws-ecs';
+import { Annotations, Construct } from '@aws-cdk/core';
+
+export class StackConfig extends Construct {
+	public readonly dockerRegistryAuthToken: string;
+
+	public readonly version: string;
+
+	public readonly launchType: LaunchType;
+
+	public readonly env: string[];
+
+	public constructor( scope: Construct, id: string ) {
+		super( scope, id );
+
+		this.dockerRegistryAuthToken = this.node.tryGetContext( 'dockerRegistryAuthToken' );
+
+		if ( !this.dockerRegistryAuthToken ) {
+			Annotations.of( this ).addError( 'Context parameter "dockerRegistryAuthToken" is required' );
+		}
+
+		this.version = this.node.tryGetContext( 'version' ) ?? '3.7.1';
```

Comment:
`3.8.0` is the latest but I think that the `latest` tag should work on our docker registry as well.